frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -341,8 +345,30 @@ const ActivitySummary = ({
           </FormItem>
         </div>
         <div className="margin-top-2">
+          <Drawer
+            triggerRef={activityReasonRef}
+            stickyHeader
+            stickyFooter
+            title="Why was this activity requested?"
+          >
+            <ContentFromFeedByTag tagName="ttahub-tta-request-option" className="ttahub-drawer--objective-topics-guidance" contentSelector="table" />
+          </Drawer>
           <FormItem
-            label="Why was this activity requested? "
+            className="margin-0"
+            customLabel={(
+              <Label className="margin-bottom-0" htmlFor="activityReason">
+                Why was this activity requested?
+                {' '}
+                <Req />
+                <button
+                  type="button"
+                  className="usa-button usa-button--unstyled margin-left-1 activity-summary-button-no-top-margin"
+                  ref={activityReasonRef}
+                >
+                  Get help choosing an option
+                </button>
+              </Label>
```

Comment:
I noticed the button is inside the <label>. Should we move it outside, or is there a reason to keep it this way?

Reply:
Yea ill move it outside.